### PR TITLE
Add code of conduct and include badge for it in README

### LINKED ISSUE #742: docs: add code of conduct
Reusable template: https://www.contributor-covenant.org/

### PR DESCRIPTION
**Pull request by kumaranvpl**
# Description

* Add code of conduct and include badge for it in README
* Fix typo

Fixes #742 

## Type of change

- [x] Documentation (typos, code examples, or any documentation updates)

## Checklist

- [x] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
- [x] I have made the necessary changes to the documentation
- [x] My changes do not generate any new warnings
- [x] Both new and existing unit tests pass successfully on my local environment by running `scripts/test-cov.sh`
- [x] I have ensured that static analysis tests are passing by running `scripts/static-anaylysis.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,11 @@
   </a>
 
   <a href="https://github.com/airtai/faststream/blob/main/LICENSE" target="_blank">
-    <img src="https://img.shields.io/github/license/airtai/faststream.png" alt="Github">
+    <img src="https://img.shields.io/github/license/airtai/faststream.png" alt="License">
+  </a>
+
+  <a href="https://github.com/airtai/faststream/blob/main/CODE_OF_CONDUCT.md" target="_blank">
```
Comment (Lancetnik) [inline README.md:44]: Not sure about another bage adding: there are too much already

Comment (kumaranvpl) [reply README.md:44]: Shall I remove it then?
Please take a look at the newly added badge before answering though - https://github.com/airtai/faststream/tree/742-docs-add-code-of-conduct

Comment (Lancetnik) [reply README.md:44]: Loots good enough. We can remove it anytime if it will be required

**Review by Lancetnik — APPROVED** [review]

**Comment by Lancetnik** [comment]
Good job, btw)

**Comment by Lancetnik** [comment]
@kumaranvpl can you add this badge to the docs index too?

**Comment by kumaranvpl** [comment]
> @kumaranvpl can you add this badge to the docs index too?

I have added it and created a new PR here - https://github.com/airtai/faststream/pull/762 @Lancetnik

**Pull request merged**